README.md

```diff
@@ -175,6 +175,8 @@ All configuration options are controlled through environment variables.
 
 * `LOG_LEVEL` - application log level, can be any of `DEBUG`, `INFO`, `WARN`, `ERROR`, defaults to `INFO`
 * `LOG_STREAM` - log destination stream, can be any of `STDOUT`, `STDERR`, defaults to `STDOUT`
+* `DEYE_PUBLISH_ON_CHANGE` - when set to `true`, the event data will only be published if it has changed compared to last readings, defaults to `false`
+* `DEYE_EVENTS_EXPIRY` - when `DEYE_PUBLISH_ON_CHANGE` is `true`, this variable defines the maximum age of a valid event list in seconds, defaults to 360 seconds
```

Comment:
I think the name `DEYE_EVENTS_EXPIRY` might be hard to understand, and for sure is not self explanatory. Event is an implementation detail, that carries the **state**, namely - an observation. My proposal is to rename it to `DEYE_PUBLISH_ON_CHANGE_MAX_INTERVAL`. WDYT?

Reply:
`DEYE_PUBLISH_ON_CHANGE_MAX_INTERVAL` is quite a long name, what about `DEYE_MAX_EVENT_INTERVAL` ?
But if you prefer the longer name, it's ok for me.

Reply:
Yes, it's a bit long, yet it's also a self-explanatory name, that is clearly related with `DEYE_PUBLISH_ON_CHANGE` flag.
Perhaps you could add a short description of this feature in the README?

Reply:
done, please have a look if the explanation makes sense